pippo-core/src/main/java/ro/pippo/core/ParameterValue.java

```diff
@@ -420,6 +421,15 @@ public Timestamp toSqlTimestamp(Timestamp defaultValue) {
         }
     }
 
+    public <T, C extends Converter<T>> T convert(Class<C> converterClass, String pattern) {
```

Comment:
Do we need this (static) method in this class? We can move the content of this method directly in Request class where we call ParameterValue#convert.
What you say?

Reply:
I think I did not understand.

Reply:
I reviewed my idea and I think that is not add value, so I gave it up.